bitnami/common/README.md

```diff
@@ -87,6 +87,8 @@ The following table lists the helpers available in the library which are scoped
 | `common.labels.standard`    | Return Kubernetes standard labels                    | `.` Chart context |
 | `common.labels.matchLabels` | Return the proper Docker Image Registry Secret Names | `.` Chart context |
 
+`common.labels.matchLabels` might include extra labels if necessary from `.Values.extraMatchLabels`. This might be necessary when a chart is used multiple times in an umbrella chart in order to ensure different stateful sets or deployments are not using overlapping match labels.
```

Comment:
We can't set this here since we are implementing the autogeneration of the READMEs and this message will disappear.

Reply:
Hi @miguelaeh, I assumed this is a very generic thing that might be required by most charts when used multiple times in an umbrella chart. But I understand the concern. Will update this PR shortly to be on a specific chart level (Redis) instead.